src/Chapter1.hs

```diff
@@ -508,7 +510,7 @@ branches because it is an expression and it must always return some value.
   satisfying the check will be returned and, therefore, evaluated.
 -}
 closestToZero :: Int -> Int -> Int
-closestToZero x y = error "closestToZero: not implemented!"
+closestToZero x y = min (abs x) (abs y)
```

Comment:
Nice idea! 💡 

However, it would return the absolute values instead of the numbers themselves.

Reply:
I will make this change in my next commit. Thank you for the catch !